workerapi/worker.thrift

```diff
@@ -0,0 +1,38 @@
+enum Status {
+  UNKNOWN = 0
+  READY = 1
+  PENDING = 2
+  FAILURE = 3
```

Comment:
I don't think we care what the error code is but we don't want to drop it either. In that light, FAILURE and SUCCESS are clearly separate states I'd think.